sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/metric/api/Tags.java

```diff
@@ -84,6 +87,26 @@ public static Tags of(Map<String, String> tags) {
         return result;
     }
 
+    /**
+     * Adds a core scope tag.
+     *
+     * @return Tags Returns the current Tags instance to support method chaining.
+     */
+    public Tags addCoreScope() {
+        tags.put(SCOPE_KEY, SCOPE_CORE);
+        return this;
+    }
+
+    /**
+     * Adds a plugin scope tag.
+     *
+     * @return Tags Returns the current Tags instance to support method chaining.
+     */
+    public Tags addPluginScope() {
```

Comment:
The specific plugin name should be used.

Reply:
done